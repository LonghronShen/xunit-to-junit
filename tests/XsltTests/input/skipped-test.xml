<?xml version="1.0" encoding="utf-8"?>
<assemblies timestamp="01/01/2018 16:40:12">
  <assembly name="Contoso.Hello.HelloTests.dll" environment="64-bit .NET Standard [collection-per-class, parallel (4 threads)]" test-framework="xUnit.net 2.3.1.3858" run-date="2018-01-01" run-time="16:40:12" total="1" passed="0" failed="0" skipped="1" time="0.090" errors="0">
    <errors />
    <collection total="1" passed="0" failed="0" skipped="1" name="Test collection for Contoso.Hello.HelloTests.EvenMoreUselessTests" time="0.000">
      <test name="Contoso.Hello.HelloTests.EvenMoreUselessTests.WhenDoWork_ThenSomeSweetJson" type="Contoso.Hello.HelloTests.EvenMoreUselessTests" method="WhenDoWork_ThenSomeSweetJson" time="0" result="Skip">
        <reason><![CDATA[I want some output man]]></reason>
      </test>
    </collection>
  </assembly>
</assemblies>