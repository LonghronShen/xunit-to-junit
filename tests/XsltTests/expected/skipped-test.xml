<?xml version="1.0" encoding="utf-8"?>
<testsuites>
  <testsuite name="Contoso.Hello.HelloTests.dll" tests="1" failures="0" skipped="1" errors="0" time="0.090" timestamp="2018-01-01T16:40:12">
    <testcase name="WhenDoWork_ThenSomeSweetJson" classname="Contoso.Hello.HelloTests.EvenMoreUselessTests" time="0">
      <skipped><![CDATA[I want some output man]]></skipped>
    </testcase>
  </testsuite>
</testsuites>